docs/decisions/characteristics/0003-security.md

```diff
@@ -1,82 +1,74 @@
 status: accepted
-date: {YYYY-MM-DD when the decision was last updated}
-deciders: {list everyone involved in the decision}
+date: 2022-11-04
+deciders: Kavya, Susmitha, Uma, Miguel, Shari
 
-# {short title of solved problem and solution}
+# Security as a Core Architectural Characteristic
 
 ## Context and Problem Statement
 
-To be successful Hey Blue! must be able to maintain trust with their Civilian and Officer community members. For Officers the drivers are privacy and safety (in particular as it relates to live location data). For Civilians the drivers are privacy and data ownership.
+To be successful Hey Blue! must be able to maintain trust with their Civilian and Officer community members. For Officers the drivers are privacy and safety (in particular as it relates to live location data). For Civilians the drivers are privacy, safety and data ownership.
 
-In this regard we must ensure the system follows Zero Trust principles, encrypts all data in motion and at rest, applies appropriate rules to the storage, access and flow of Personally Identifiable Information (PII), allows members to control visibility of and revoke their PII, and conforms to all required regulatory standards (such as GDPR).
+## Decision Drivers
 
-<!-- This is an optional element. Feel free to remove. -->
+- Trust - While not called out explicitly in requirements, it is necessary to have ongoing engagement. The trust is built on keeping data secure (encryted), controlling access
+- Regulatory - The system must conform to all required regulatory standards, such as GDPR.
 
-## Decision Drivers
+The following decision drivers are directly derived from the requirements and kickoff artefacts from Hey Blue!
 
-- {decision driver 1, e.g., a force, facing concern, …}
-- {decision driver 2, e.g., a force, facing concern, …}
-- … <!-- numbers of drivers can vary -->
+### Officers
 
-## Considered Options
+- Safety - Officers need to be able to turn off location services
+- Safety - Officers need to have their location services automatically turn off after 15 minutes of activity.
+- Safety/Privacy - Must be able to have an interaction without being located.
+- Privacy - Officers need to be able to hide their profiles
```

Comment:
I'm thinking Civilians might also would like to have an option of hiding their profiles when they need a break ?

Reply:
ignore this. the civilian section covers.